src/Microsoft.OData.Edm/Microsoft.OData.Edm.csproj

```diff
@@ -86,7 +86,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <AdditionalFiles Include="PublicAPI\$(TargetFramework)\**" />
+    <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Shipped.txt" />
+    <AdditionalFiles Include="PublicAPI/$(TargetFramework)/PublicAPI.Unshipped.txt" />

```

Comment:
Are there other files in the `PublicAPI/$(TargetFramework)`.

Reply:
If at some point these files get renamed then someone has to update them here... Also, if more files are added to the directory then they have to be added individually?

Reply:
These are the files expected by the Public API analyzer. See: https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md#conditional-api-differences

Reply:
I thought having `<AdditionalFiles Include="PublicAPI\$(TargetFramework)\**" /> ` was sufficient for it to work

Reply:
But the files have to exist with those names.